### PR TITLE
fix for Issue1916  Pgr_point_t.

### LINKED ISSUE #1916: Making compilation faster  include/c_types/pgr_point_t.h
Refer to #1893 for instructions and details.

### LINKED ISSUE #1926: Making compilation faster include/c_types/restriction_t.h
Refer to #1893 for instructions and details.

### PR DESCRIPTION
**Pull request by ManasviGoyal**
Fixes #1916

Changes proposed in this pull request:

-changes to pgr_point_t..h
-adjusting code based on changes on structure Pgr_point_t.

@pgRouting/admins

### CODE REVIEW

**Comment by ManasviGoyal** [comment]
@cvvergara pls check and so sorry for the incorrect references. pls tell if i can fix that

File: include/c_common/pgr_point_input.h [hunk 1]
```diff
@@ -29,8 +29,7 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 /* for size-t */
 #include <stddef.h>
 
-#include "c_types/pgr_point_t.h"
-
+typedef struct Pgr_point_input_t Pgr_point_input_t;
```
Comment (cvvergara) [inline include/c_common/pgr_point_input.h:32]: The actions will point if it is wrong (by sight it is), probably you need to change to
```
typedef struct Pgr_point_t Pgr_point_t;
```
btw, maximum 2 commits.
```
git fetch upstream
git rebase -i upstream/develop
```
Use fixup, to put the extra commits into one (help shows on the same file it opens)

**Review by cvvergara — CHANGES_REQUESTED** [review]

**Comment by cvvergara** [comment]
@ManasviGoyal Now you need to try fixup the commits

**Comment by ManasviGoyal** [comment]
@cvvergara i tried to fixup but it now added other commits which were made before me. what is it that i am doing wrong?

**Pull request closed without merging**